README.md

```diff
@@ -451,6 +452,11 @@ This option defaults to `false`.
 * Default: `false`
 * Use partial appends in the transmuxer and segment loader
 
+##### liveRangeSafeTimeDelta
+* Type: `boolean`,
+* Default: `false`
```

Comment:
This is a number that defaults to the SAFE_TIME_DELTA

Reply:
Done 👍🏻